backend/database/factories/CompanyFactory.php

```diff
@@ -14,19 +15,21 @@
 use Internships\Models\Specialization;
 use Internships\Models\Submission;
 use Internships\Models\User;
-use Smknstd\FakerPicsumImages\FakerPicsumImagesProvider;
+use Internships\Services\LogoGenerator;
 
 class CompanyFactory extends Factory
 {
     public function definition(): array
     {
-        fake()->addProvider(new FakerPicsumImagesProvider(fake()));
+        $name = deep_copy(fake()->company());
```

Comment:
I think deep_copy is unnecessary in that case

Reply:
Ah, that's right. 
It's a leftover from when I was trying to figure out why LogoGenerator was sometimes creating logos with different initials than the company name (which was caused by not setting it up in the configure() method).